src/components/tabs/tabbed_content/tabbed_content.tsx

```diff
@@ -58,7 +58,7 @@ export type EuiTabbedContentProps = CommonProps &
      * Use this prop if you want to control selection state within the owner component
      */
     selectedTab?: EuiTabbedContentTab;
-    size?: EuiTabsSizes;
+    size?: 's' | 'm' | 'l' | 'xl';
```

Comment:
You can import the type from `src/components/tabs/tabs.tsx`:

```suggestion
    size?: EuiTabsSizes;
```